src/types/music.ts

```diff
@@ -18,3 +18,11 @@ export type Playlist = {
     name: string;
     id: number;
 };
+
+export type Artist = {
```

Comment:
albums should be an Album[] type, not Track[], Album will contain tracks, cover and other data. Also you forget about linkedArtists: Artist[] field, that should be optional. Check Artist Screen UI ticket to create correct Artist type

Reply:
It fixed in other PR